third_party/istio-1.1-latest

```diff
@@ -1 +1 @@
-istio-1.1.3
\ No newline at end of file
```

Comment:
<!--whitespace-bot-->
```suggestion
istio-1.1.7

```

Reply:
This is not a file, but a a symlink generated by the command "ln -s istio-1.1.7 istio-1.1-latest".